research/flamby/single_node_trainer.py

```diff
@@ -95,7 +95,7 @@ def validate(self, val_metric_mngr: MetricManager) -> None:
             for input, target in self.val_loader:
                 input, target = input.to(self.device), target.to(self.device)
 
-                preds = self.model(input)
+                preds = {"predictions": self.model(input)}
```

Comment:
Small fix that must have been missed when we migrated to dictionaries for predictions at some point.